lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -627,12 +636,16 @@ def write(chunk)
                 log_name: log_name,
                 resource: group_level_resource,
                 labels: group_level_common_labels,
+                partial_success: @partial_success,
                 entries: entries)
             entries_count = entries.length
 
             # TODO: RequestOptions
```

Comment:
Is this TODO covered by the call below?

Reply:
Ah, this `TODO` is really old. Just removed it.